resources/views/account/account.blade.php

```diff
@@ -25,7 +25,20 @@
                         @if($user->countryCode)
                             <td>{{$user->countryCode}}-{{$user->phoneNumber}}</td>
                         @else
-                            <td>{{$user->phoneNumber}}</td>
+                            <!-- <td>{{$user->phoneNumber}}</td> -->
+                            @php
+                            $new = strval($user->phoneNumber);
+                            $phone = ' ';
+                            $count = 0;
+                            for($i = 0; $i < 10; $i++){
+                                $phone .= $new[$i];
+                                if(($i+1) % 3 == 0 && $count < 2){
+                                    $phone .= '-';
+                                    $count++;
+                                }
+                            }
+                            @endphp
+                            <td>{{$phone}}</td>
```

Comment:
The code looks good overall, but could you put this as a function in the User Model instead? Name it something like `formattedPhoneNumber()`. Then in the blade here, you can just do: `{$user->formattedPhoneNumber}}` to get the formatted phone number. This makes the code more reusable, and keeps PHP out of the blade templates.